src/components/hooks/use-browser-resize.ts

```diff
@@ -12,7 +12,7 @@ export const useBrowserResize = (maxSize) => {
     }
 
     useEffect(() => {
-        if (!isBrowser()) return false
+        if (!isBrowser()) return () => false
```

Comment:
```suggestion
        if (!isBrowser()) return
```
we can just return here.

Reply:
1) Please also type the input on line 5.
2) All hooks should end in `.tsx` extension as they are react specific.